android/local.properties

```diff
@@ -0,0 +1 @@
+sdk.dir=C:\\Users\\Asus\\AppData\\Local\\Android\\Sdk
```

Comment:
@offler9 This part, you should handle by using somekind of env variable next time. Because another member will stuck using this windows path when try to build this project